lib/cli.go

```diff
@@ -41,207 +41,207 @@ func CLI(program, version string, flags Flags) error {
 	}
 
 	// All of these flags can be accessed in your GAPI implementation with
-	// the `c.Parent().Type` and `c.Parent().IsSet` functions. Their own
+	// the `c.Lineage()[1].Type` and `c.Lineage()[1].IsSet` functions. Their own
```

Comment:
It's especially dope when you also port the comments. I do this too. Not being sarcastic. :thumbsup: